solr/core/src/test/org/apache/solr/cloud/MoveReplicaHDFSTest.java

```diff
@@ -50,8 +50,8 @@ public static void teardownClass() throws Exception {
       HdfsTestUtil.teardownClass(dfsCluster);
     } finally {
       dfsCluster = null;
-      System.setProperty("solr.hdfs.blockcache.blocksperbank", "512");
```

Comment:
This was introduced recently for the Hadoop 3 upgrade. Copy/paste error but definitely causing some of the new test failures.